### ISSUE TITLE
the app only said "a heart atack ocurred"

### ISSUE THREAD

**Issue by xken2x** [cSploit/android#533]
I try install and use the app bit when i update The core (is necesary for use the app) the app download, check And decompress the cobre, but the app only show the mensaje and i cannot do nothing

**Comment by tux-mind** [comment]
Hi @xken2x , what is TVE app ?

**Comment by xken2x** [comment]
Excuse me, the selfcorrection change what i try to write XD
The error happens in 3 devices than i have, the app download and install the core, but finally only said "a heart atack ocurred" and i dont know what do.

**Comment by pahapoika** [comment]
Hey,
First read this https://github.com/cSploit/android/wiki/How-to-open-an-issue

1.What version are you using? You have the latest 1.6.3?  https://github.com/cSploit/android/releases
 
2.Have you tried to download the nightly? Maybe it works 
http://rootbitch.cc/csploit/cSploit-nightly.apk
3.Your device model 
4.

Can you post a pic or maybe video? It could help.

**Comment by pahapoika** [comment]
Sorry, 
i closed

**Comment by pahapoika** [comment]
Sorry,
I closed this accidentally. Im on car now on phone and the car jumps when i was typing.

So 
4.whats your device ROM and version 
5.Whether or not you are using Xposed or any Xposed modules. (By the way, if you are-- don't)
6. logcat (very important!) taken while the application was running and the issue happened.
learn what it is and how to use it >http://forum.xda-developers.com/showthread.php?t=1726238

**Comment by pahapoika** [comment]
@xken2x

**Comment by Briareos12** [comment]
Do you use kingroot ?

**Comment by xken2x** [comment]
yes, i root the phones with kingroot, but after i root, try of change the app by superuser, but dont allow me to change it.
excuse my bad english :P

**Comment by Briareos12** [comment]
Did you try supersu method ?

**Comment by IwraStudios** [comment]
You should try supersu kingroot is known to cause trouble with cSploit

**Comment by IwraStudios** [comment]
sorry, wrong button
